packages/xo-server-test/src/_config.js

```diff
@@ -3,6 +3,8 @@ import path from 'path'
 
 /* eslint-env jest */
 
+jest.setTimeout(600000)
+
```

Comment:
Remove.

You can do this in another PR if you think it's relevant :slightly_smiling_face: